### PR TITLE
Convex Object Break example: Enhancements

### PR DESCRIPTION
**Pull request by yomboprime**
Add 3D audio to the `webgl_physics_convex_break` example (3 audio samples, I have limited it to max 8 sound sources per sample) The sounds are played at the position of physical impacts.

Also made some refactoring:
- Added linear and angular velocity to debris just after the collision (better simulation)
- Added gui with more scenes to select from. Enable audio button, reset scene button.
- Physics world destroy code (to re-create another scene)
- Less temporary Ammo vectors used.
- Added more comments.

Live demo: https://yomboprime.github.io/ConvexBreakSound3D/webgl_physics_convex_break.html
Ah, don't forget to activate the sound in the GUI.

### CODE REVIEW

**Comment by yomboprime** [comment]
Nobody tried this? @mrdoob @Mugen87 I think it sounds cool. Or do you find it not interesting, or overcomplicated? Please let me know.

**Comment by Mugen87** [comment]
Not sure if we should add sounds effects to the examples apart from the actual web audio demos. I think I would remove them and just focus on the physics. The example is already very complex compared to other ones.

**Comment by yomboprime** [comment]
> Not sure if we should add sounds effects to the examples apart from the actual web audio demos. I think I would remove them and just focus on the physics. The example is already very complex compared to other ones.

Yes, I think you're right.

**Comment by yomboprime** [comment]
I removed the 3D sound. I think the other changes are worth it.

**Comment by yomboprime** [comment]
>     * Added linear and angular velocity to debris just after the collision (better simulation)

Note: This means that the debris get the velocities from the parent if it was moving before breaking apart. It does not mean that they get extra velocity from the collision, thing that could also be simulated.

File: examples/webgl_physics_convex_break.html [hunk 1]
```diff
@@ -482,61 +728,75 @@
...
 				// If no point has contact, abort
-				if ( ! contact ) {
-					continue;
-				}
+				if ( ! contact ) continue;
 
 				// Subdivision
 
-				var fractureImpulse = 250;
+				const fractureImpulse = 230;
```
Comment (Mugen87) [inline examples/webgl_physics_convex_break.html]: If possible use just `var` in this example.

Comment (yomboprime) [reply examples/webgl_physics_convex_break.html]: Done!

**Comment by yomboprime** [comment]
Should I commit here the changes in Ammo init code from https://github.com/mrdoob/three.js/pull/16100 ?

**Comment by mrdoob** [comment]
> Should I commit here the changes in Ammo init code from #16100 ?

Yes please 🙏

**Comment by yomboprime** [comment]
> > Should I commit here the changes in Ammo init code from #16100 ?
> 
> Yes please pray

Done 😊

**Comment by yomboprime** [comment]
I don' know how to resolve merge conflicts. How should I proceed?

**Comment by yomboprime** [comment]
New PR: #16126

**Pull request closed without merging**